apps/spinner/src/App.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "zarkit/react";
+import "./App.scss";
+import Spinner from "./spinner"
+import ChanceWheel from "./chance-wheel"
+
+
+function App(){
+
+const [state, setState] = useState({
```

Comment:
naming a state `"state"` is not a good idea, you can separate the values of object and store as separate proper named states.